README.md

```diff
@@ -404,7 +404,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 
 * [20 Ruby on Rails interview questions and answers from CareerRide.com](http://www.careerride.com/ruby-on-rails-interview-questions.aspx)
 * [9 Essential Ruby on Rails Interview Questions from Toptal](http://www.toptal.com/ruby-on-rails/interview-questions)
-* [A list of common questions with answers ask during interview of ruby on rails job](https://github.com/rishiip/ruby-on-rails-interview-questions)
+* [A list of common questions with answers ask during an interview of ruby on rails job](https://github.com/rishiip/ruby-on-rails-interview-questions)
```

Comment:
Could you change to "A list of common questions with answers to ask during an interview of ruby on rails job"